modules/4337/test/erc4337/ERC4337ModuleNew.spec.ts

```diff
@@ -119,6 +123,147 @@ describe('Safe4337Module - Newly deployed safe', () => {
       expect(await ethers.provider.getBalance(safe.address)).to.be.eq(ethers.parseEther('0'))
     })
 
+    it('should revert when signature length contains additional bytes - EOA signature', async () => {
```

Comment:
I _think_ we keep the tests in `ERC4337ModuleExisting.spec` matching the ones here.

Reply:
Not worth it.